liblangutil/SourceReferenceFormatter.cpp

```diff
@@ -210,5 +214,11 @@ void SourceReferenceFormatter::printErrorInformation(ErrorList const& _errors)
 
 void SourceReferenceFormatter::printErrorInformation(Error const& _error)
 {
-	printExceptionInformation(SourceReferenceExtractor::extract(m_charStreamProvider, _error));
+	SourceReferenceExtractor::Message message =
+		SourceReferenceExtractor::extract(
+			m_charStreamProvider,
+			_error,
+			Error::errorSeverity(_error.type())
+		);
```

Comment:
This is fine with our style but a bit unusual :) We'd usually do it like this:
```suggestion
	SourceReferenceExtractor::Message message = SourceReferenceExtractor::extract(
		m_charStreamProvider,
		_error,
		Error::errorSeverity(_error.type())
	);
```